src/ci-line-chart.js

```diff
@@ -256,16 +263,17 @@ const vis = {
     Array.from(pivotFieldNames).forEach((group, idx) => {
       // Filter data to the current group
       const groupData = d3data.filter((d) => d.pivot === group);
+      const groupLabel = group != 'NONE' ? ` - ${group}` : '';
 
       cfg.data.datasets.push({
-        label: `Lower-${group}`,
+        label: 'Lower' + groupLabel,
         data: groupData,
         fill: 1,
         parsing: {
           yAxisKey: 'CI_left'
         },
-        borderColor: idx === 0 ? 'rgba(0,0,200,1)' : 'rgba(200,0,0,1)',
-        backgroundColor: idx === 0 ? 'rgba(0,0,200,0.5)' : 'rgba(200,0,0,0.5)',
+        backgroundColor: config[group] + "33",
+        borderColor: config[group],
```

Comment:
use configured color and adjust opacity

Reply:
Great! We may want to add something to ensure the configured color is hex format. RGB is technically accepted by the color picker (though the user would have to type/paste it in manually).